src/main/java/org/jitsi/jicofo/bridge/Bridge.java

```diff
@@ -49,6 +50,49 @@
     private static final ColibriStatsExtension EMPTY_STATS
         = new ColibriStatsExtension();
 
+    /**
+     * A conservative estimate of the average sum of egress + egress packet rate
+     * (in pps) of a video stream flowing through the bridge. Note that the term
+     * "video stream" comes from {@link ColibriConferenceImpl#trackVideoChannelsAddedRemoved},
+     * so it's equivalent to a participant.
+     */
+    private static final int AVG_VIDEO_STREAM_PACKET_RATE_PPS;
+
+    /**
+     * This is the worst case scenario that the bridge must be able to handle
+     * in a 100-peeps call with 20 local senders and 5 local receivers.
+     */
+    private static final double MAX_TOTAL_PACKET_RATE_PPS;
+
+    static
+    {
+        MaxPacketRateCalculator packetRateCalculator = new MaxPacketRateCalculator(
```

Comment:
So this is intended to reflect a "worst case" packet calculator?

Reply:
Yes